src/Equinor.ProCoSys.IPO.WebApi/Permissions.cs

```diff
@@ -5,5 +5,6 @@ public class Permissions
         public const string COMMPKG_READ = "COMMPKG/READ";
         public const string PROJECT_READ = "PROJECT/READ";
         public const string MCPKG_READ = "MCPKG/READ";
+        public const string LIBRARY_FUNCTIONAL_ROLE_READ = "LIBRARY_FUNCTIONAL_ROLE/READ";
```

Comment:
Could we get away with just using LIBRARY/READ? for reading library data.

Reply:
I don't think so as there's no permission called just 'LIBRARY'